packages/components/organisms/f-checkout/src/components/Address.vue

```diff
@@ -115,13 +119,22 @@ export default {
     },
 
     methods: {
+        ...mapActions('checkout', [
+            'updateFulfilmentAddress'
+        ]),
         /*
         * Returns true if `field` has been touched and if it is still empty
         * The $dirty boolean changes to true when the user has focused/lost
         * focus on the input field.
         */
         isFieldEmpty (field) {
             return this.$v.addressValidations[field].$dirty && !this.$v.addressValidations[field].required;
+        },
+        /*
+        * Dispatches map action `updateFulfilmentAddress` to update input fields values in vuex
+        */
+        updateAddressDetails (key, event) {
+            this.updateFulfilmentAddress({ [key]: event })
```

Comment:
I think this will pass the whole event object to this key now? I think what you want is: `[key]: key` now?